.gitignore

```diff
@@ -22,3 +22,4 @@ apps/blockchain/config/dev.secret.exs
 .erlang_cookie
 
 !/**/.gitkeep
+/.vscode
```

Comment:
I think adding editor specific files to gitignore is bad practice

Reply:
Is there a downside here? If you use the editor, you won't accidentally add editor-specific files, and if you don't, nothing will happen. Maybe we just group editor-specific ignores and put them at the bottom?

Reply:
@ayrat555 , @willmeister I removed it (though I don't see much problems with having it) and added it in my local _.git/info/exclude_ file.